Controller/MediaAdminController.php

```diff
@@ -77,10 +77,11 @@ public function listAction(Request $request = null)
         $datagrid->setValue('context', null, $context);
 
         // retrieve the main category for the tree view
-        $category = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        $rootCategory = $this->container->get('sonata.classification.manager.category')->getRootCategory($context); 
```

Comment:
Aha, i see it now, that's a nice bug for StyleCI :-) In chrome the line is cut off to early, so you dont see the issue. Thanks for pointing that out ^^